source/ways-of-working/new-component/feature-flags.html.md.erb

```diff
@@ -30,8 +30,13 @@ End to end Process:
 ### Accessing LaunchDarkly
 
 Developers automatically get access by being part of 'DTS CFT Developers' or 'DTS SDS Developers', non developers can get access with the 'DTS LaunchDarkly' group.
-see [onboarding](../../onboarding/person/index.html) process to request access.
-[URL](https://app.launchdarkly.com/)
+see [onboarding](../../onboarding/person/index.html) process to request access. You will need to login from [myapps.microsoft.com](https://myapps.microsoft.com).
+If you don't see it and you are in the correct groups then make sure you are in the 'CJS Common Platform' tenant, click your profile picture in the top right and it should be at the top.
+If you aren't then click 'Switch organisation' and select the correct one.
```

Comment:
isn't it 'Switch Directory'

Reply:
nope not in myapps:

![image](https://user-images.githubusercontent.com/21194782/157451372-6c12dbb6-df7d-40b8-b0b4-6d9f2da9f51f.png)

it is in the portal though 👅